### PR TITLE
Improve rtmroswtf

### PR DESCRIPTION
**Pull request by 130s**
- [x] Clarify which tarball to send back
- [x] Add rosversion for all ROS packages
- [x] Add `rospack find` to see where the specific (or all of the above ROS) packages are installed
- [x] `git status` for all packages (to see the possible changes made by the users locally)

### CODE REVIEW

**Comment by 130s** [comment]
Added `git status for all packages (to see the possible changes made by the users locally)`

**Comment by 130s** [comment]
Please review @k-okada 

If commits need squashed I can do that.

File: hrpsys_tools/scripts/rtmroswtf.sh [hunk 1]
```diff
@@ -12,12 +43,30 @@ FILENAME_LOG_ALL=/tmp/rtmros_diagnosisinfo_all_`date +"%Y%m%d-%H%M%S"`.tgz
...
+  printf "%s $i " 2>&1 | tee -a ${FILENAME_LOG_COMMANDS}
+  p_underscore=$(echo ${i} | awk '{print $1}' | sed 's/ros-\([a-zA-Z]*\)-//' | tr '-' '_');
+  printf "%s ${p_underscore} " 2>&1 | tee -a ${FILENAME_LOG_COMMANDS}
+  rospackfind_result=$(rospack -q find ${p_underscore})
+  printf "%s ${rospackfind_result} " 2>&1 | tee -a ${FILENAME_LOG_COMMANDS}
+  echo "${rospackfind_result}" | grep -o '[^/]*$' | xargs rosversion 2>&1 | tee -a ${FILENAME_LOG_COMMANDS}
+  if [[ -z "$rospackfind_result" ]]; then
+    continue;
+  fi
+  # git status
```
Comment (k-okada) [inline hrpsys_tools/scripts/rtmroswtf.sh]: need to run git status on source package, not package found by `dpkg -s | grep ros...`, how about using `rospack list` and check if that directory have .git before running git status

Comment (130s) [inline hrpsys_tools/scripts/rtmroswtf.sh]: Must be already addressed in https://github.com/130s/rtmros_common/commit/2671e374046a73163022349fd3426033c4cb8864

**Comment by 130s** [comment]
What about https://github.com/130s/rtmros_common/pull/2? @k-okada

**Pull request merged**